### PR TITLE
Form validation

### LINKED ISSUE #81: Sanatize form data and add captcha
**Describe the bug**
We need to sanitize all form data and set up add captcha to form.

### PR DESCRIPTION
**Pull request by DarshitJain04**
### Description
Added validation for form fields. 
No extra dependencies are required for this change.

FormValidation

Here's a snapshot of Company Registration form where the pincode field has been validated.

Fixes #81 

### Type of Change:

- Code

**Code/Quality Assurance Only**
- Bug fix
- New feature

### Checklist:

- [x] My PR follows the style guidelines of this project

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by Tanuj22** [comment]
@DarshitJain04 I am seeing previous commits

File: src/accounts/validators.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+
+def validate_gpa(value):
+    reg = re.compile(r'^\d\.?\d*$')
+    if not reg.match(value):
+        raise ValidationError('This field should contain only numeric values')
+    return value
+
+
+def validate_phone(value):
```
Comment (Tanuj22) [inline src/accounts/validators.py:20]: We should remove this. As we other care in what format they enter the phone no. Like I can begin my phone number with 0 if I want to and have an 11 digit number

**Comment by DarshitJain04** [comment]
> Great job! Just some minor changes. Move the validators `check_file_size`  and `regex_validators` ( also `validator_fn` into the new file that you created 

Okay I'll make these changes!
So should I just keep a check if the phone number is numeric only ?

**Comment by Tanuj22** [comment]
> > Great job! Just some minor changes. Move the validators `check_file_size`  and `regex_validators` ( also `validator_fn` into the new file that you created
> 
> Okay I'll make these changes!
> So should I just keep a check if the phone number is numeric only ?

Should we remove it? I may enter a number like +91 9876543210. Unless a better regex is available removing it is the best option ig

**Comment by DarshitJain04** [comment]
> > > Great job! Just some minor changes. Move the validators `check_file_size`  and `regex_validators` ( also `validator_fn` into the new file that you created
> > 
> > Okay I'll make these changes!
> > So should I just keep a check if the phone number is numeric only ?
> 
> Should we remove it? I may enter a number like +91 9876543210. Unless a better regex is available removing it is the best option ig 

Okay I'll remove it

**Pull request merged**